mmv1/third_party/terraform/services/clouddeploy/resource_clouddeploy_custom_target_type_test.go

```diff
@@ -80,6 +80,14 @@ resource "google_clouddeploy_custom_target_type" "custom-target-type" {
           ref = "main"
         }
       }
+      include_skaffold_modules {
+        configs = ["my-config3"]
+        google_cloud_build_repo {
+          repository = "projects/example/locations/us-central1/connections/git/repositories/example-repo"
```

Comment:
After thinking about it more, although it works as is. In the past, the service team has gotten around to validating input values.

So as it stands this would cause 100% failures in the event that they add a validation which is likely to happen.

Because of this we'll want to have an actual valid repository to prevent this from happening in the future.

Reply:
What kind of validation does the service team do?

The referenced resource (the GCB repo) only needs to exist when a Release for a Target that references a CustomTargetType is used.

Reply:
Circling back on this @BBBmau